lib/src/patterns/facade/Facade.dart

```diff
@@ -32,11 +31,11 @@ class Facade implements IFacade
    *
    * -  Throws [MultitonErrorFacadeExists] if instance for this Multiton key has already been constructed.
    */
-  Facade( String key )
-  {
-    if ( instanceMap[ key ] != null ) throw new MultitonErrorFacadeExists();
-    initializeNotifier( key );
-    instanceMap[ multitonKey ] = this;
+  Facade(String key) {
+    if (instanceMap.containsKey(key)) throw MultitonErrorFacadeExists();
+    _multitonKey = key;
```

Comment:
Change `_multitonKey` back to `multitonKey`.

Reply:
Its done.